pgspecial/iocommands.py

```diff
@@ -132,6 +132,7 @@ def _index_of_file_name(tokenlist):
     "\\copy",
     "\\copy [tablename] to/from [filename]",
     "Copy data between a file and a table.",
+    case_sensitive=False,
```

Comment:
I see this added but not used. Did you check in all of your changes?

Reply:
The existing behavior of `copy` doesn't need to change - just the way it's registered as a backslash-command via `@special_command`.

I think setting `case_sensitive=False` is all it takes to make this library recognize `\COPY` as `\copy`. The test added in this PR passes for me; if I `git checkout @^ -- pgspecial/iocommands.py` to undo my change in this file, then run `py.test`, it fails with `pgspecial.main.CommandNotFound: Command not found: \COPY`.

I've removed the extra import. I'd originally come down with a case of JSbrain and added `arg_type=PARSED_QUERY, hidden=False, case_sensitive=False` here, as though default arguments only worked for parameters listed _after_ the last non-default argument, and while I eventually remembered how Python works I did not remember to remove that import. I don't know why `black .` and the pre-commit hook didn't catch it.